src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -65,6 +85,11 @@ internal async Task<bool> ScheduleDurableTaskEvents(OrchestrationInvocationResul
             {
                 try
                 {
+                    this.traceHelper.ProcessingOutOfProcPayload(
+                        taskHub: "TBD",
```

Comment:
I wasn't sure how to access the taskhub within the OOProcShim. I figured it's a necessity for identifying the orchestration instance in kusto, right?